src/framework/uicomponents/qml/Muse/UiComponents/internal/StyledMenu.qml

```diff
@@ -46,6 +49,13 @@ MenuView {
 
     signal loaded()
 
+    onNotationViewNavigationSectionChanged: function() {
+        menuNavPanel.section = root.notationViewNavigationSection
+    }
+    onNavigationOrderStartChanged: function() {
+        menuNavPanel.order = root.navigationOrderStart
+    }
```

Comment:
This isn't very idiomatic QML: instead of setting the property imperatively to a fixed value, a binding (inside the `NavigationPanel` would be better:
```qml
    section: root.navigationSectionOverride ?? content.navigationSection
    order: root.navigationOrderStart
```

(which would remove the need for the addition of the `id: menuNavPanel` line)